tools/generate-record-row.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""
+Generate wheel metadata RECORD file row for given file.
+
+Usefull in cases where you want to postprocess the wheel archive e.g. include
+additional libraries after the wheel has been build.
+"""
```

Comment:
Provide a Usage: line with what parameters are expected

Reply:
I added argparse which will provide `--help` and usage info.